docs/example-iam-policy.json

```diff
@@ -25,15 +25,7 @@
       "Resource": [
         "arn:aws:ec2:*:*:volume/*",
         "arn:aws:ec2:*:*:snapshot/*"
-      ],
-      "Condition": {
-        "StringEquals": {
-          "ec2:CreateAction": [
-            "CreateVolume",
-            "CreateSnapshot"
-          ]
-        }
-      }
```

Comment:
As far as I am aware, there is no condition to allow CreateTags on ModifyVolume (correct me if I'm wrong), so if the customer was to use the example policy as is, tag modification will not work and they won't be authorized to perform the CreateTags operation on a volume.

Reply:
The change here is removing, not adding any additional permissions.

> so if the customer was to use the example policy as is, tag modification will not work 

The existing managed policy already allows the `ec2:CreateTags` and `ec2:DeleteTags` actions on volume and snapshot resources: https://docs.aws.amazon.com/aws-managed-policy/latest/reference/AmazonEBSCSIDriverPolicy.html

Reply:
Yes but for `ec2:CreateTags` it only allows it for the specified conditions `"Condition" : {
        "StringEquals" : {
          "ec2:CreateAction" : [
            "CreateVolume",
            "CreateSnapshot"
          ]
        }
      }` This will not allow it for when we are modifying a volume, which is why I had to remove those conditions